File: go/docmost/models/file_task.go
package models

// FileTask from OpenAPI.
type FileTask struct {
	Id *string `json:"id,omitempty"`
	FileName *string `json:"fileName,omitempty"`
	FileExt *string `json:"fileExt,omitempty"`
	FilePath *string `json:"filePath,omitempty"`
	FileSize *string `json:"fileSize,omitempty"`
	Type_ *string `json:"type,omitempty"`
	Source *string `json:"source,omitempty"`
	Status *string `json:"status,omitempty"`
	ErrorMessage *string `json:"errorMessage,omitempty"`
	CreatorId *string `json:"creatorId,omitempty"`
	SpaceId *string `json:"spaceId,omitempty"`
	WorkspaceId *string `json:"workspaceId,omitempty"`
	CreatedAt *string `json:"createdAt,omitempty"`
	UpdatedAt *string `json:"updatedAt,omitempty"`
}

File: c#/src/Docmost.Sdk/Models/FileTask.cs
namespace Docmost.Sdk.Models;

public class FileTask
{
    [System.Text.Json.Serialization.JsonPropertyName("id")]
    public string? Id { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("fileName")]
    public string? FileName { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("fileExt")]
    public string? FileExt { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("filePath")]
    public string? FilePath { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("fileSize")]
    public string? FileSize { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("type")]
    public string? Type { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("source")]
    public string? Source { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("status")]
    public string? Status { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("errorMessage")]
    public string? ErrorMessage { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("creatorId")]
    public string? CreatorId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("spaceId")]
    public string? SpaceId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("workspaceId")]
    public string? WorkspaceId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("createdAt")]
    public string? CreatedAt { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("updatedAt")]
    public string? UpdatedAt { get; set; }

}
